examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -16,15 +16,36 @@
    "cell_type": "code",
```

Comment:
<p>This plot is actually <code>az.plot_energy</code> the cell can be reduced to a single line (could be done in a follow up PR)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>